presto-docs/src/main/sphinx/release/release-0.143.rst

```diff
@@ -22,6 +22,8 @@ Hive Changes
 ------------
 
 * Fix native memory leak when reading or writing gzip compressed data.
+* Fix predicate pushdown where complex expressions are not applied when
```

Comment:
I'd rephrase as "Fix performance regression due to complex expressions not being applied when pruning partitions"